src/javascript/ast-value.ts

```diff
@@ -30,12 +30,27 @@ function literalToValue(literal: estree.Literal): LiteralValue {
 /**
  * converts unary to string
  */
-function unaryToValue(unary: estree.UnaryExpression): string {
-  const argValue = expressionToValue(unary.argument);
-  if (argValue === undefined) {
-    return;
+function unaryToValue(unary: estree.UnaryExpression): LiteralValue {
```

Comment:
Good point. Also updated some of the other comments, most of which are of limited usefulness now that we've got good estree typings.